python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/defs_additional_paths.schema.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+type: dict
+$defs:
+  additional_paths:
+    type: dict
+    keys:
+      install:
+        type: bool
+      install_ecmp_primary:
+        type: bool
+      receive:
+        type: bool
+      send:
```

Comment:
There is option to add `prefix-list` also under all values of `send` in `address-family ipv4/ipv6` -

```
s1-leaf2(config-router-bgp-af)#neighbor 1.1.1.2 additional-paths send any ?
  prefix-list  Apply the configuration only to routes matching the prefix list
  <cr>         

s1-leaf2(config-router-bgp-af)#neighbor 1.1.1.2 additional-paths send any prefix-list ?
  WORD  Prefix list

s1-leaf2(config-router-bgp-af)#neighbor 1.1.1.2 additional-paths send any prefix-list pf1 ?
  <cr>  

s1-leaf2(config-router-bgp-af)#neighbor 1.1.1.2 additional-paths send any prefix-list pf1 
s1-leaf2(config-router-bgp-af)#show active
router bgp 6500
   address-family ipv4
      neighbor 1.1.1.2 additional-paths send any prefix-list pf1
```

Reply:
Added.

Reply:
This is for `neighbors[].additional_paths` only not for `bgp.additional_paths`

Reply:
Fixed.